downstreamadapter/worker/producer/kafka_dml_producer_test.go

```diff
@@ -1,197 +0,0 @@
-// Copyright 2022 PingCAP, Inc.
```

Comment:
This should not be removed completely, this test should be irrelevant to the sarama

Reply:
confluent-kafka-go doesn't support mock the producer. https://github.com/confluentinc/confluent-kafka-go/issues/875#issuecomment-1297386052

Reply:
we have to find a way to test the DML producer, no need to depend on the mocking producer, we can implement one